nav_msgs/Odometry robot_pose
navigation/TwistArray obstacles_velocities
visualization_msgs/Marker potential_field


